### PR TITLE
[DOCS-8850] adding more info to SNMP traps about enabling logs

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?

[DOCS-8850](https://datadoghq.atlassian.net/browse/DOCS-8850)

There have been multiple support requests where it wasn't clear the user needed to enable logs to receive SNMP traps data. Broke out this section to multiple steps to make that more clear , and added a screen shot on how to find SNMP traps data in the Logs Explorer once enabled.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-8850]: https://datadoghq.atlassian.net/browse/DOCS-8850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 1]
```diff
@@ -17,10 +17,15 @@ Datadog Agent v7.37+ supports listening for SNMP Traps, enabling you to set up [
 
 ## Configuration
 
-To enable listening for SNMP Traps, add the following to `datadog.yaml`:
+1. The SNMP Trap functionality sends data as logs to Datadog. To receive this data, ensure that log collection is enabled by adding the following configuration to your `datadog.yaml` file.
 
 ```yaml
 logs_enabled: true # Traps are forwarded as logs and can be found in the Log Explorer with a `source:snmp-traps` query.
+```
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/snmp_traps.md:24]:
~~~suggestion
   ```yaml
   logs_enabled: true # Traps are forwarded as logs and can be found in the Log Explorer with a `source:snmp-traps` query.
   ```
~~~

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 2]
```diff
@@ -17,10 +17,15 @@ Datadog Agent v7.37+ supports listening for SNMP Traps, enabling you to set up [
...
+
+2. In addition to enabling log collection on your Agent, to enable listening for SNMP Traps, add the following to your `datadog.yaml` file:
+
+```yaml
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/snmp_traps.md:28]:
```suggestion
   ```yaml
```

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 3]
```diff
@@ -48,7 +53,12 @@ network_devices:
       privProtocol: "AES" # choices: DES, AES (128 bits), AES192, AES192C, AES256, AES256C
 ```
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/snmp_traps.md:54]:
~~~suggestion
   ```
~~~

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 4]
```diff
@@ -48,7 +53,12 @@ network_devices:
...
 
-**Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
+  **Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
+
+3. Once configured, search in the [Log Explorer][5] for `source:snmp_traps` to locate SNMP trap data:
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/snmp_traps.md:58]:
```suggestion
3. Search for `source:snmp_traps` in the [Log Explorer][5] to locate SNMP trap data:
```

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 5]
```diff
@@ -48,7 +53,12 @@ network_devices:
       privProtocol: "AES" # choices: DES, AES (128 bits), AES192, AES192C, AES256, AES256C
 ```
 
-**Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
+  **Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
```
Comment (hestonhoffman) [inline content/en/network_monitoring/devices/snmp_traps.md:56]:
```suggestion
   **Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
```

**Review by hestonhoffman — APPROVED** [review]
I left some formatting edits to try and line the code blocks and note up under the instruction. The YAML block might require 3 extra spaces before each line to get it working (I couldn't do that in the web UI).

File: content/en/network_monitoring/devices/snmp_traps.md (hunk 5 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/snmp_traps.md:56]: Thanks let me try this! I was struggling with the markdown to get the formatting right as well :)

**Pull request merged**